### ISSUE TITLE
Formulas are not being calculated

### ISSUE THREAD

**Issue by hugomelo** [randym/axlsx#355]
When I create a simple spreadsheet using axlsx 2.0.1 gem and open the generated file with libreoffice (didn't test with Excel), the formulas are not calculated by default except by the first line. 
It used to work in an earlier version I used before I upgraded to 2.0.1

The code I used to reproduce it was:

require 'axlsx'
p = Axlsx::Package.new
wb = p.workbook
wb.add_worksheet(:name => 'spread') do |sheet|
  sheet.add_row ['name', 'value', 'quantity', 'total']
  sheet.add_row ['n', 2, 7, '=B2 * C2']
  sheet.add_row ['n', 2, 71, '=B3 * C3']
  sheet.add_row ['n', 21, 7.1, '=B4 * C4']
end
p.serialize '/tmp/as.xlsx'

**Comment by fbandrey** [comment]
@hugomelo last formula works for me, but I can't understand what you want to recieve by ```=B2C2``` and ```=B3C3``` formulas, they broken. Checking under Microsoft Excel for Mac 2011.

**Comment by hugomelo** [comment]
Hi @fbandrey, github was removing the *. I've added spaces and now it shows that.
I kind of discovered what happened. The problem wasn't with axlsx, but with libreoffice and gnumeric which stopped calculating by default the formulas. I need to manually ask those tools to recalc all the formulas to show it correctly. As a workaround, I've used the formula_value attribute to prepopulate the cells. I am closing this ticket as it's not a bug.
BTW, It seems there's something missing here. When I open the spreadsheet created by axlsx, ask it to recalculate and save. Then, I reopen it, the values are shown correctly.
I wonder if we could make this calculation step on axlsx, so the different tools can open it in lazy mode and show the exact values of the formulas
